endpoints/openrtb2/amp_auction_test.go

```diff
@@ -372,6 +374,71 @@ func TestGDPRConsent(t *testing.T) {
 	}
 }
 
+func TestSetConsentedProviders(t *testing.T) {
+
+	sampleBidRequest := &openrtb2.BidRequest{}
+
+	testCases := []struct {
+		description         string
+		inAdditionalConsent string
+		inBidRequest        *openrtb2.BidRequest
```

Comment:
Nitpick: Could these be renamed to `givenAdditionalConsent` and `givenBidRequest`?

Reply:
Done